image/image_test.go

```diff
@@ -249,3 +254,68 @@ func TestCheckFileInfo(t *testing.T) {
 		t.Error("sha512 should not be matched")
 	}
 }
+
+func TestUncompressedContentSize(t *testing.T) {
+	contentSize := int64(2 * 1024)
+
+	dir, err := createTestDir("testDir1", contentSize)
+	if err != nil {
+		t.Fatalf("Can't create test dir: %v", err)
+	}
+
+	defer os.Remove(dir)
+
+	tarFile1 := path.Join(workDir, "archive1.tar")
+
+	if output, err := exec.Command("tar", "-C", dir, "-cf", tarFile1, "./").CombinedOutput(); err != nil {
+		t.Fatalf("Can't create tar archive: %v", fmt.Sprintf("error: %v, code: %v", string(output), err))
```

Comment:
Why do you need printf in printf?
`t.Fatalf("Can't create tar archive: message: %s, err: %v", string(output), err)`

Reply:
part of this message was copy-pasted from another UT

Reply:
Done